pages/docs/faq.mdx

```diff
@@ -2,35 +2,50 @@
 layout: 'docs'
 title: 'FAQ'
 metaTitle: 'FAQ'
-metaDescription: 'Mach mit!'
+metaDescription: 'Frequently Asked Questions zur Plattform'
```

Comment:
How about "Häufig gestellte Fragen zur Plattform"?

Reply:
like it --> will update the metadDeascription within the next push